build_tools/ci/cpu_comparison/run.py

```diff
@@ -1777,6 +1796,26 @@ def __init__(self):
                 additional_labels=["I8UKernel"],
             )
         )
+        self.register(
+            Matmul(
+                64,
+                64,
+                64,
+                "bf16",
+                "f32",
+                name_suffix="4rows_8cols_npu4_pack_peel_4_level_tiling_ukernel",
+                use_ukernel=True,
+                tile_pipeline="pack-peel-4-level-tiling",
+                run_on_target=["npu4"],
+                aie_compilation_flags=[
+                    "--iree-amdaie-num-rows=4",
+                    "--iree-amdaie-num-cols=8",
+                    "--iree-amd-aie-additional-peano-opt-flags=-O3",
+                    "--iree-amdaie-enable-function-outlining=True",
+                ],
+                use_chess=True,
```

Comment:
```suggestion
```

Reply:
The `use_chess=True,` is still there?

Reply:
Did you try removing it? I don't think it needs to be there as even when using peano for general compilation, the ukernel will still be compiled using chess.

Reply:
And not all ukernel tests have `use_chess=True`: https://github.com/nod-ai/iree-amd-aie/blob/7a059cdd2a40f99921e1283c6a340c2a7cf0973e/build_tools/ci/cpu_comparison/run.py#L1771

Reply:
Yep, I did.
It lead to the failure only in that e2e test where it didn't find some `mm_primitive` in the `.o` (that's what I can recollect from the CI) - but I can reconfirm the same and revert.

Reply:
Here's the error after removing `use_chess=True` :-
```
ld.lld: error: undefined symbol: me_primitive::control_rnd
>>> referenced by mm_npu4.o
>>>               /home/shark-strix-1/workspace/iree-amd-aie/test_matmuls/matmul_64_64_64_bf16_f32_4rows_8cols_npu4_pack_peel_4_level_tiling_ukernel_ukernel/mm_npu4.o:(void matmul_vectorized_bf16_f32<2u, 8u, 1u, 8u, 8u, 8u>(bfloat16 const*, unsigned int, bfloat16 const*, unsigned int, float*, unsigned int))
```

Reply:
Interesting, might not be implemented in peano yet I guess, let's keep `use_chess=True` then. It might be good to track this error down a bit further and create an issue on the llvm-aie repo if it's indeed unsupported.